packages/docx-io/src/lib/mammoth.js/browser-demo/index.html

```
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)